### DISCUSSION TITLE
Missing "domesticWaterDemand.nc" file for tutorial 4

### DISCUSSION THREAD

**Discussion by Fah-K** [iiasa/CWatM#85] [Q&A]
Hello, 

I am following tutorial 4, however, I received an error message as the following

Error 212: Netcdf map stacks:
In  "domesticWaterDemandFile"
path: ..\..\..\CWatM-Earth-30min\landsurface\waterDemand exists
but filename: domesticWaterDemand.nc does not
file name extension can be .nc4 or .nc

I tried to search in the GitHub repository, however, I could not find any. Would you mind helping me locate the file or drop a file for download? 

Thank you!

**Comment by mikhailsmilovic** [comment]
Guten Morgen, we suggest using the settings file [settings_CWatM_template_30min.ini](https://github.com/iiasa/CWatM-Earth-30min/blob/main/settings_CWatM_template_30min.ini) within the [CWatM-Earth-30min](https://github.com/iiasa/CWatM-Earth-30min) repository. This settings file points to updated demand files 🏭
